components/RoundButtons.js

```diff
@@ -1,131 +1,251 @@
 import React from "react";
-import { Feather } from '@expo/vector-icons';
-import { View, StyleSheet, TouchableOpacity, Text, Dimensions} from "react-native";
-
+import { Feather } from "@expo/vector-icons";
+import {
+  View,
+  StyleSheet,
+  TouchableOpacity,
+  Text,
+  Dimensions,
+  Image,
+} from "react-native";
 
 //Element po którym pseudo dziedziczy, dla reacta się na to mówi "zawieranie"
-export const RoundButton = ({navigation, destination, icon, title, color}) => {
-    return(
-        <TouchableOpacity
-        onPress={() => {
-            navigation.navigate(destination);
-          }}
-        >
+export const RoundButton = ({
+  navigation,
+  destination,
+  icon,
+  title,
+  color,
+}) => {
+  return (
+    <TouchableOpacity
+      onPress={() => {
+        navigation.navigate(destination);
+      }}
+    >
+      <View style={styles.buttonBox}>
+        <View style={[styles.button, colors[color]]}>
+          <Feather
+            name={icon}
+            size={Dimensions.get("window").width * 0.18}
+            color="white"
+          />
+        </View>
+        <Text style={[styles.buttonText, textColors[color]]}>{title}</Text>
+      </View>
+    </TouchableOpacity>
+  );
+};
 
-            <View style={styles.buttonBox}>
-                <View style={[ styles.button, colors[color] ]}>
-                    <Feather name={icon} size={Dimensions.get('window').width * 0.18} color="white" />
-                </View>
-                <Text
-                style={[ styles.buttonText, textColors[color] ]}
-                >
-                    {title}
-                </Text>
-            </View>
+// CustomRoundButton to przycisk który potrzebuje specjalnej grafiki (takiej której nie ma w "@expo/vector-icons")
+export const CustomRoundButton = ({
```

Comment:
Jest to ciekawy pomysł, i sam CustomRoundButton zostawię, ale z powodów skalowania wolę pozostać przy korzystaniu z zwyczajnego RoundButtona. Skorzystamy po prostu z grafik podobnych do tych na projekcie graficznym.